tensorflow/core/kernels/reduction_gpu_kernels.cu.h

```diff
@@ -348,20 +348,8 @@ __global__ __launch_bounds__(1024) void ColumnReduceKernel(
   value_type sum = initVal;
   if (row < num_rows && col < num_cols) sum = in[row * num_cols + col];
 
-    // 1D array necessary due to bug in CUDA 9 compiler.
-    // TODO(nluehr) revert to 2D array when compiler is ready.
-    // This is to mimic the following, but without constructors:
```

Comment:
I don't think your changes resolves the TODO.  It mentions 2D array, and then says it's explicitly creating a raw array to avoid constructors.  Your change replaces with a 1D array, that presumably does call constructors.

@nluehr do you have any more context here?

Reply:
I think it is better to let @nluehr resolve it or update the affected version comments instead.

Reply:
Thanks for raising this to my attention. Since we've dropped support for CUDA 9.x in https://github.com/tensorflow/tensorflow/pull/59717, it's time to clean these up. I'll go ahead and file a PR shortly.

Reply:
Thanks @nluehr!